### DISCUSSION TITLE
pages/about

### DISCUSSION THREAD

**Discussion by giscus** [dreamhunter2333/dreamhunter-blog#13] [General]
# pages/about

你指尖跃动的电光，是我此生不变的信仰

https://dreamhunter2333.com/pages/about.html

**Comment by dreamhunter2333** [comment]
任何问题都可以留言

**Comment by itcuihao** [comment]
`6`

**Reply by dreamhunter2333** [reply]
你好

**Reply by itcuihao** [reply]
很高兴认识你

**Comment by ZeroWang86** [comment]
我申请了发邮件权限，但是等了一个月也没审核通过。我需要这个功能，不会发很多邮件，占用内存也不多。可以审核通过下吗？极客速邮的用户名是1440137520@qq.com，谢谢大佬！

**Reply by dreamhunter2333** [reply]
> 我申请了发邮件权限，但是等了一个月也没审核通过。我需要这个功能，不会发很多邮件，占用内存也不多。可以审核通过下吗？极客速邮的用户名是[1440137520@qq.com](mailto:1440137520@qq.com)，谢谢大佬！

你好，极客速邮并不是我搭建的临时邮箱，
我 GitHub 的示例项目地址为 https://mail.awsl.uk

**Comment by 8rootting** [comment]
大佬，shang.（       ）.xyz 这个网关能帮忙开个账号吗 或者能否发个联系方式

**Reply by dreamhunter2333** [reply]
你好，不清楚你发的网站是谁的，不是我这的

**Comment by lord16801** [comment]
grok自助充值怎么对接.大佬

**Reply by dreamhunter2333** [reply]
你好，不清楚grok自助充值对接啥
